assets/app/views/modals/process-template.html

```diff
@@ -0,0 +1,21 @@
+<div class="modal-resource-process">
+  <div class="modal-body">
+    <h1>You are adding a Template</h1>
+    <p>Please select type of actions you want to do with your defined Template:</p>
+
+    <div>
+      <input type="checkbox" ng-model="templateOptions.process"/>
+      <strong>Process the template</strong>
+      <span id="helpBlock" class="help-block">This will create an instance of the things defined in the template, you will have an opportunity to fill out any parameters the template defines.</span>
+    </div>
+    <div>
+    <input type="checkbox" ng-model="templateOptions.add"/>
+    <strong>Add template</strong>
+    <span id="helpBlock" class="help-block">This will save the tamplate and it available to others who have access to this project.</span>
+    </div>
+  </div>
```

Comment:
maybe...

**Add Template**

What would you like to do?

[x] Process template
Create the objects defined in the template. You will have an opportunity to fill in template parameters.

[x] Save template
Save the template to the project.  This will make the template available to anyone who can view the project.